docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -236,11 +236,12 @@ Finally, there are a couple of updates to Django's documentation to make:
    the "Features deprecated in A.B" heading.
 
 #) Add an entry in the deprecation timeline (``docs/internals/deprecation.txt``)
-   under the ``A.B+2`` version describing what code will be removed.
+   under the ``A.B+2`` (or B+3 for features deprecated in the version after an
```

Comment:
`A.B+3` will never exist; it's really `A+1.0` for features deprecated in the version following an LTS. And often `A.B+2` will actually be a release in the `A+1` series, too. I'm not sure if we can rely on people to figure this out themselves? It wouldn't be that hard to just explicitly list the three possibilities here (`A.0` -> `A+1.0`, `A.1` -> `A+1.0`, `A.2` -> `A+1.1`).